lib/between_meals/repo/svn.rb

```diff
@@ -91,6 +96,23 @@ def valid_ref?(ref)
       rescue
         raise Changeset::ReferenceError
       end
+
+      private
+
+      def parse_status(changes)
+        # http://svnbook.red-bean.com/en/1.0/re26.html
+        changes.lines.map do |line|
+          case line
+          when (/^([\w ])\w?\s+(\S+)$/)
```

Comment:
That's what @elthariel pointed out above, and I agree with. We care about files changes (first column), and not attribute changes (second column). But space in the first column and whatever in second column is considered a file modification.